packages/playwright-core/src/server/chromium/crBrowser.ts

```diff
@@ -343,7 +343,7 @@ export class CRBrowserContext extends BrowserContext {
   override async _initialize() {
     assert(!Array.from(this._browser._crPages.values()).some(page => page._browserContext === this));
     const promises: Promise<any>[] = [super._initialize()];
-    if (this._browser.options.name !== 'electron' && this._browser.options.name !== 'clank') {
+    if (this._browser.options.name !== 'electron' && this._browser.options.name !== 'clank' && !this._options.keepDefaultDownloadBehavior) {
```

Comment:
- What about other browsers? This option should work in all of them.
- Perhaps it makes sense to replace `options.acceptDownloads` with an enum `downloads: 'accept' | 'deny' | 'internal-browser-default'`?